### PR TITLE
@orta -> hide metadata view while rotating artwork view

### PR DESCRIPTION
**Pull request by 1aurabrown**
fixes https://github.com/artsy/eigen/issues/465

Pretty basic solution - fade it out quickly at the beginning of the rotation, fade it back in at the end.

I'm not sure how to accomplish this using the `coordinator`'s `animateAlongsideTranstion`

### CODE REVIEW

File: Artsy/Classes/Views/ARArtworkMetadataView.m [hunk 1]
```diff
@@ -39,58 +36,58 @@ - (instancetype)initWithArtwork:(Artwork *)artwork andFair:(Fair *)fair
     self.artworkPreviewActions = previewActionsView;
     self.artworkDetailView = artworkDetailView;
 
-    UIView *imageContainer = [[UIView alloc] init];
-    UIView *textContainer = [[UIView alloc] init];
+    UIView *left = [[UIView alloc] init];
```
Comment (orta) [inline Artsy/Classes/Views/ARArtworkMetadataView.m]: I see the point you're going for, but this feels a bit like writing "left" as a CSS class, you sure you want to do that?

Comment (1aurabrown) [inline Artsy/Classes/Views/ARArtworkMetadataView.m]: I felt like it was easier to read if you were trying to interact with it
from another object. For example, I was trying to figure out which view it
was that I wanted to fade in and out, and had to do a lot of reading to
determine which one was the right column. I don't feel particularly
strongly either way though.

On Fri, May 22, 2015 at 4:23 PM, Orta <notifications@github.com> wrote:

> In Artsy/Classes/Views/ARArtworkMetadataView.m
> <https://github.com/artsy/eigen/pull/477#discussion_r30927645>:
>
> > @@ -39,58 +36,58 @@ - (instancetype)initWithArtwork:(Artwork *)artwork andFair:(Fair *)fair
> >      self.artworkPreviewActions = previewActionsView;
> >      self.artworkDetailView = artworkDetailView;
> >
> > -    UIView *imageContainer = [[UIView alloc] init];
> > -    UIView *textContainer = [[UIView alloc] init];
> > +    UIView *left = [[UIView alloc] init];
>
> I see the point you're going for, but this feels a bit like writing "left"
> as a CSS class, you sure you want to do that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/artsy/eigen/pull/477/files#r30927645>.
>



-- 

[A.]


Laura Brown

Artsy <http://artsy.net/>

laura@artsy.net <caroline@artsy.net>

Comment (1aurabrown) [inline Artsy/Classes/Views/ARArtworkMetadataView.m]: the one time that github has ever formatted an email comment correctly for me

Comment (orta) [inline Artsy/Classes/Views/ARArtworkMetadataView.m]: :D

Comment (orta) [inline Artsy/Classes/Views/ARArtworkMetadataView.m]: I don't feel too strongly, sounds good to me.

**Pull request merged**